vector.ms

```diff
@@ -45,7 +45,11 @@ list.clamp = function(min, max)
 end function
 
 list.cross = function(n)
-    return [-self[1] * n, self[0] * n]
+    if n isa list then
+        return self[0] * n[1] + self[1] * n[0]
+    else
+        return [-self[1] * n, self[0] * n]
+    end if
```

Comment:
I'm confused about what list.cross is doing.  It doesn't look like a cross product.  But, let's discuss on Discord.